bunsen/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>com.google.fhir.analytics</groupId>
     <artifactId>root</artifactId>
-    <version>0.1.0-SNAPSHOT</version>
+    <version>0.2.7-SNAPSHOT</version>
```

Comment:
Just a suggestion to see if we can optimise the `version` to be present at root level as a property and all the modules inherit from this global property.

Reply:
Resolving this as it is related to the first PR.